src/common/HeartbeatMap.cc

```diff
@@ -71,13 +71,13 @@ bool HeartbeatMap::_check(const heartbeat_handle_d *h, const char *who, time_t n
 
   was = h->timeout;
   if (was && was < now) {
-    ldout(m_cct, 1) << who << " '" << h->name << "'"
+    ldout(m_cct, 0) << who << " '" << h->name << "'"
 		    << " had timed out after " << h->grace << dendl;
     healthy = false;
   }
   was = h->suicide_timeout;
   if (was && was < now) {
-    ldout(m_cct, 1) << who << " '" << h->name << "'"
+    ldout(m_cct, 0) << who << " '" << h->name << "'"
```

Comment:
This promotion is not of much sense as we are soon going to fire the assert below, right?